Realm/RLMResults.h

```diff
@@ -24,53 +24,54 @@ NS_ASSUME_NONNULL_BEGIN
 @class RLMObject, RLMRealm, RLMNotificationToken;
 
 /**
- RLMResults is an auto-updating container type in Realm returned from object
- queries.
+ `RLMResults` is an auto-updating container type in Realm returned from object
+ queries. It represents the results of the query in the form of a collection of objects.
 
- RLMResults can be queried with the same predicates as RLMObject and RLMArray
- and you can chain queries to further filter query results.
+ `RLMResults` can be queried using the same predicates as `RLMObject` and `RLMArray`,
+ and you can chain queries to further filter results.
 
- RLMResults always reflect the current state of the Realm on the current thread,
+ `RLMResults` always reflect the current state of the Realm on the current thread,
  including during write transactions on the current thread. The one exception to
  this is when using `for...in` fast enumeration, which will always enumerate
  over the objects which matched the query when the enumeration is begun, even if
  some of them are deleted or modified to be excluded by the filter during the
  enumeration.
 
- RLMResults are initially lazily evaluated, and only run queries when the result
+ `RLMResults` are lazily evaluated, and only run queries when the result
```

Comment:
"initially" is correct here; once you access it the results are eagerly kept up-to-date for as long as you continue to access the results after every write transaction.